drivers/entropy/Kconfig.psa_crypto

```diff
@@ -8,6 +8,9 @@ config ENTROPY_PSA_CRYPTO_RNG
 	depends on DT_HAS_ZEPHYR_PSA_CRYPTO_RNG_ENABLED
 	select ENTROPY_HAS_DRIVER
 	select PSA_WANT_GENERATE_RANDOM
+	select SSF_PSA_CRYPTO_SERVICE_ENABLED if SOC_SERIES_NRF54HX
+	depends on ZCBOR || !(SOC_SERIES_NRF54HX || SOC_SERIES_NRF92X)
+	depends on !DCACHE || !(SOC_SERIES_NRF54HX || SOC_SERIES_NRF92X)
```

Comment:
Simplify to `!DCACHE if (SOC_SERIES_NRF54HX || SOC_SERIES_NRF92X)`?

Reply:
This is not valid Kconfig AFAIK :/ But I should have removed the 92X here, so still it gives me something :)

Reply:
Are you sure? :thinking:

Reply:
Yep, give it a try.
Ooooor, ask chatgpt:
 
![image](https://github.com/user-attachments/assets/98ef5377-3b88-4f5f-82e0-0153a6423c93)

Reply:
Okay then you could combine both `depends on` you added into (given you are removing the 92X) `!(SOC_SERIES_NRF54HX && (!ZCBOR || DCACHE))`.

Reply:
I don't find it better I am afraid. It has double negations which I try to avoid in general :/